data/card_definitions.json

```diff
@@ -5813,6 +5813,367 @@
 		"NOTEEND": "LEO CARDS END"
 	},
 
+	{
+		"NOTE": "LINNE CARDS START",
+		"id": "linne_divineblaze",
+		"type": "ultra",
+		"display_name": "Divine Blaze",
+		"force_cost": 0,
+		"gauge_cost": 3,
+		"range_min": 1,
+		"range_max": 2,
+		"speed": 6,
+		"power": 3,
+		"armor": 0,
+		"guard": 0,
+		"effects": [
+			{
+				"timing": "hit",
+				"effect_type": "push",
+				"amount": 3,
+				"and": {
+					"effect_type": "gain_advantage"
+				}
+			},
+			{
+				"timing": "during_strike",
+				"condition": "last_turn_was_strike",
+				"effect_type": "powerup",
+				"amount": 4
+			}
+		],
+		"boost": {
+			"boost_type": "continuous",
+			"force_cost": 0,
+			"cancel_cost": -1,
+			"display_name": "Culling (Demise Razor)",
+			"effects": [
+				{
+					"timing": "hit",
+					"effect_type": "opponent_discard_random",
+					"amount": 1,
+					"and": {
+						"effect_type": "gain_advantage"
+					}
+				},
+				{
+					"timing": "now",
+					"effect_type": "strike"
+				}
+			]
+		}
+	},
+
```

Comment:
Can you only leave extra newlines between cards when it is different characters?